packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -140,7 +151,22 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 	// These are ordered based on speed and reliability for ease of debugging
 	const frameworkTests: Record<string, FrameworkTestConfig> = {
 		astro: {
-			expectResponseToContain: "Hello, Astronaut!",
+			expectResponseToContain: "Welcome to Astro",
+			testCommitMessage: true,
+		},
+		docusaurus: {
+			expectResponseToContain: "Dinosaurs are cool",
+			testCommitMessage: true,
+		},
+		gatsby: {
+			skip: true,
```

Comment:
Any reason to skip here instead of just `quarantine: true`?

Reply:
my understanding of the quarantine is that it has frameworks that are very very flaky but it's still worth running them, in this case this seems to practically always fail, so I don't think there is a real benefit in keep running it, no?

if you prefer it in the quarantine I'm happy to change it that way, but again if it's practically broken I feel like it might just be skipped? 🤷

Reply:
that's fair, let's skip it